dojo/jira_link/views.py

```diff
@@ -26,12 +26,23 @@
 
 
 @csrf_exempt
-def webhook(request):
-    # Webhook shouldn't be active if jira isn't enabled
+def webhook(request, secret=None):
```

Comment:
`@require_http_methods(["POST"])` maybe? What do we want to allow through that webhook?

Reply:
add. good idea, although it makes testing it harder ;-)